front/src/PositionAwareApp/AssetsViewer/index.tsx

```diff
@@ -39,8 +39,8 @@ const CARSTATION_ICON = L.icon({
   popupAnchor: [12, -24]
 });
 
-const BIKESTATION_ICON = L.icon({
-  iconUrl: "/bikestation_picto.svg",
+const BICYCLESTATION_ICON = L.icon({
+  iconUrl: "/bicyclestation_picto.svg",
```

Comment:
Was it worth it?

Reply:
Not really, but just to be consistent until the end